.claude/skills/beopsuny/scripts/maintenance/check_law_updates.py

```
@@ -0,0 +1,399 @@
+    if root is None:
+        return None
+
+    return {
+        "name": root.findtext(".//법령명_한글", "") or root.findtext(".//법령명", ""),
+        "promul_date": root.findtext(".//공포일자", ""),
+        "enforce_date": root.findtext(".//시행일자", ""),
+        "revision_type": root.findtext(".//제개정구분명", ""),
+    }
+
+
+def get_recent_amendments(oc_code: str, from_date: str, to_date: str = None) -> Tuple[list, bool]:
```

Ruff (0.14.10)

PEP 484 prohibits implicit `Optional`

Convert to `T | None`

(RUF013)